fastlane/lanes/localization.rb

```diff
@@ -375,13 +387,19 @@
   # bundle exec fastlane download_translations
   #####################################################################################
   lane :download_translations do
-    # For now WordPress and Jetpack use the same GlotPress project and share the same `strings.xml`
-    # (the Jetpack-dedicated one, https://translate.wordpress.com/projects/jetpack/apps/android/, is empty anyway for now).
+    # WordPress strings
     android_download_translations(
       res_dir: File.join('WordPress', 'src', 'main', 'res'),
-      glotpress_url: 'https://translate.wordpress.org/projects/apps/android/dev/',
-      locales: ALL_LOCALES,
-      lint_task: 'lintWordpressVanillaRelease' # TODO: Should we adapt this?
+      glotpress_url: APP_SPECIFIC_VALUES[:wordpress][:glotpress_appstrings_project],
+      locales: WP_APP_LOCALES,
+      lint_task: 'lintWordpressVanillaRelease'
```

Comment:
TODO: Remove the call to the lint tasks (which runs them locally on the Release Manager's Mac, which can take quite a long time) once we finally migrate the "Release Build" jobs from CircleCI to Buildkite and can make the "Release Build" jobs in Buildkite `depend_on` the Buildkite lint job.

The migration of those jobs to Buildkite was expected to be done way sooner which is why those have still be there for a while, but hopefully I plan to work on the migration of those to Buildkite very soon — which will allow me to save a lot of time during releases by not having to wait for two lint tasks to finish running on my Mac before CI starts (which often meant that I couldn't do any other work on WPAndroid while waiting for it to finish running)